src/components/Aframe/my-loader.js

```diff
@@ -298,130 +298,118 @@ AFRAME.registerComponent("myloader", {
     }
   },
 
```

Comment:
run `npm run lint` - the formatting on this file looks inconsistent in places

Reply:
thanks!